packages/ndla-notion/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ndla/notion",
-  "version": "0.2.62",
+  "version": "0.3.0",
```

Comment:
Versjonsnummer her settes ved publisering. Må nesten reverte denne endringa, men eg ser at det er breakingfunksjonalitet her så vi må hugse å sette ny versjon ved publisering.

Reply:
Egentlig er det vel ikkje breaking for customCSS er jo ikkje noko som må være med, men det skader jo ikkje å bumpe versjonen.